docs/docsite/rst/porting_guides/porting_guide_core_2.11.rst

```diff
@@ -9,7 +9,7 @@ This section discusses the behavioral changes between ``ansible-base`` 2.10 and
 
 It is intended to assist in updating your playbooks, plugins and other parts of your Ansible infrastructure so they will work with this version of ``ansible-core``.
 
-We suggest you read this page along with the `ansible-core Changelog for 2.11 <https://github.com/ansible/ansible/blob/stable-2.11/changelogs/CHANGELOG-v2.11.rst>`_ to understand what updates you may need to make.
+We suggest you read this page along with the `ansible-core Changelog for 2.11 <https://github.com/ansible/ansible/blob/devel/changelogs/CHANGELOG-v2.11.rst>`_ to understand what updates you may need to make.
```

Comment:
(This is because we haven't branched stable-2.11 yet, it will need to be changed back once we do)